src/System.Security.Permissions/src/System/Security/HostProtectionException.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace System.Security
+{
+    public partial class HostProtectionException : System.SystemException
+    {
+        public HostProtectionException() { }
+        // protected HostProtectionException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
+        public HostProtectionException(string message) { }
+        public HostProtectionException(string message, System.Exception e) { }
+        public HostProtectionException(string message, System.Security.Permissions.HostProtectionResource protectedResources, System.Security.Permissions.HostProtectionResource demandedResources) { }
```

Comment:
For any exceptions we create like this, the constructors should pass the relevant information down to the base exception, e.g. instead of:
```C#
public HostProtectionException(string message) { }
```
doing:
```C#
public HostProtectionException(string message) : base(message) { }
```